wallet/addresses.go

```diff
@@ -237,7 +248,15 @@ func (w *Wallet) nextAddress(op errors.Op, persist persistReturnedChildFunc, acc
 		if err != nil {
 			return nil, err
 		}
-		alb.cursor++
+
+		if account == udb.VotingAddrAccount {
```

Comment:
This looks incorrect, the logic should be the same for both BIP0044 accounts derived from the seed, and the voting account created by the imported pubkey